inject/src/main/java/io/micronaut/context/DefaultBeanContext.java

```diff
@@ -2462,7 +2462,7 @@ protected void processParallelBeans(List<BeanDefinitionProducer> parallelBeans)
                         try {
                             initializeEagerBean(beanDefinition);
                         } catch (Throwable e) {
-                            LOG.error("Parallel Bean definition [" + beanDefinition.getName() + MSG_COULD_NOT_BE_LOADED + e.getMessage(), e);
+                            LOG.error("Parallel Bean definition [{}{}{}", beanDefinition.getName(), MSG_COULD_NOT_BE_LOADED, e.getMessage(), e);
```

Comment:
Should the log message here contain a closing bracket?
```suggestion
                            LOG.error("Parallel Bean definition [{}{}{}]", beanDefinition.getName(), MSG_COULD_NOT_BE_LOADED, e.getMessage(), e);
```